src/composables/translateParser.ts

```diff
@@ -0,0 +1,30 @@
+import { TOptions } from 'i18next'
+import { useTranslation } from 'i18next-vue'
+
+/**
+ * Translate legacy _V3 text
+ * eg: from getLegendHtml, server returns html to be translated with angular directive "<h4 translate>Rigoles d&#39;érosion identifiées</h4>"
+ * @returns
+ */
+export function useTranslateParser() {
+  const { t } = useTranslation()
+
+  const translate = (htmlContent: string, options?: TOptions | string) => {
```

Comment:
rename to luxTranslate ?

Is there any good practice in use of arrow functions vs. function declaration ?
Instinctively, I find a function declaration with keyword 'function' more readable inside a class-like structure like here.

Reply:
> Is there any good practice in use of arrow functions vs. function declaration ? Instinctively, I find a function declaration with keyword 'function' more readable inside a class-like structure like here.

There are no good practices, arrow functions may be helpful when using `this` context, removing the confusion to what it refers at the moment the function is called. Arrow function have been introduced in es6, so it is a modern way to declare function.